packages/config-ui/demo/entry.jsx

```diff
@@ -166,6 +166,7 @@ class RawContainer extends React.Component {
                 two={{ label: 'two', value: 'two' }}
               />
               <NChoice
+                showVertical={true}
```

Comment:
This is a demo page, so we should present both options (horizontal and vertical).